src/lib.rs

```diff
@@ -198,7 +198,7 @@ mod v5;
 #[cfg(all(windows, feature = "winapi"))]
 mod winapi_support;
 
-use crate::std::{fmt, str};
+use crate::std::{convert::TryInto, fmt, str};
```

Comment:
For consistency, only import modules

Reply:
Ok, switched to just importing std::convert, and using `convert::TryFrom::try_from()` instead of `.try_from()` (which, AAUI, you can't do without importing the trait itself).

Reply:
Yup it’s done intentionally for traits to prevent any type’s implicit methods interfering...

Reply:
@kinggoesgaming Hmm, they'll only conflict with other traits imported with conflicting methods. If a type has an inherent method then it'll be chosen over any trait ones.